content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
+type: "article"
+description: "A guide on authenticating to the Chainguard Registry to get images"
+date: 2023-03-21T15:10:16+00:00
+lastmod: 2023-03-21T15:10:16+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 500
+toc: true
+---
+
```

Comment:
Is there a short description of the Registry we can add at the top to help with context for the reader and help get the page found via search?

Reply:
Is this still needed after we have the new registry overview page?